.github/workflows/run_benchmarks_main.yml

```
@@ -0,0 +1,76 @@
+name: Run benchmarks main
+
+on:
+  push:
+    branches:
+      - main
```

To test the whole flow, we can add the current branch and remove before merging.

```suggestion
    branches:
      - main
      # todo: remove the branch after testing
      - start-storing-benchmarks-data
```